packages/react-docsite-components/src/components/PageHeader/PageHeader.module.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../styles/common';
+
+.root {
+  width: calc(100% - #{$SideRail-width});
```

Comment:
Would be good to check what happens with the dropdown on mobile. (The site may totally be a mess on mobile right now but I do see some styling for things like hiding the side rail, which would also mess with the dropdown position.)

Reply:
Hmm this definitely comes out looking funny on the main controls page, at least on a smaller desktop screen. 
![image](https://user-images.githubusercontent.com/5864305/99111233-96cdd900-25a0-11eb-941a-30c36b6ec711.png)

I'm also not seeing it even on a specific control page.
![image](https://user-images.githubusercontent.com/5864305/99111285-afd68a00-25a0-11eb-8732-8e074619edd9.png)

Reply:
You should now be seeing it in every component page as well, although I still need to address the styling.